.github/workflows/tagging.yml

```diff
@@ -1,6 +1,9 @@
 name: Create Git and Docker tags
 
-on: deployment_status
+on: 
+  #deployment_status
```

Comment:
In the spirit of Dave's comments, we should remove this comment

Reply:
Removed.